### PR TITLE
create core concepts and update tutorial in getting started guide

### PR DESCRIPTION
**Pull request by droumis**
127 0 0 1_5500_docs__build_html_getting_started_installation html (1)

127 0 0 1_5500_docs__build_html_getting_started_build_dashboard html (2)

127 0 0 1_5500_docs__build_html_getting_started_core_concepts html (2)

### CODE REVIEW

**Comment by philippjfr** [comment]
This looks fabulous, great work! I'll go ahead and merge if you're happy with it.

**Comment by Hoxbro** [comment]
Use `bash` instead of `console`

**Comment by droumis** [comment]
@philippjfr , there are still some TODO's in here. Want me to remove them before the merge?

File: docs/getting_started/build_dashboard.md [hunk 1]
```diff
@@ -343,36 +352,28 @@ targets:
...
-        height: 350
-      - type: hvplot
-        kind: hist
-        y: body_mass_g
-        responsive: true
-        height: 350
+        height: 300
+      - type: table
+        show_index: false
+        height: 300
```
Comment (philippjfr) [inline docs/getting_started/build_dashboard.md:363]: Can we also update the image?

```suggestion
        height: 300
        theme: midnight
```

Comment (Hoxbro) [reply docs/getting_started/build_dashboard.md:363]: Just noting that this will also need an update to the `:emphasize-lines`

**Review by Hoxbro — APPROVED** [review]

File: docs/getting_started/build_dashboard.md (hunk 1 above)
Comment (droumis) [reply docs/getting_started/build_dashboard.md:363]: done

**Pull request merged**